make/gen.cmake

```diff
@@ -332,15 +374,15 @@ function(N_TEMPLATE)
       add_file_target(FILE ${SRC_WITH_PREFIX} GENERATED DEPENDS ${SRC})
 
       list(APPEND OUTPUTS ${SRC_WITH_PREFIX})
-
       if(${N_TEMPLATE_APPLY_V2X})
         get_filename_component(V2X_NAME ${SRC_WITH_PREFIX} NAME_WE)
-        v2x(NAME ${V2X_NAME} SRCS ${SRC_WITH_PREFIX})
+        v2x(NAME ${V2X_NAME} SRCS ${SRC_WITH_PREFIX} GENERATEDXMLDEPS ${GEN_XML_DEPS} GENERATEDVERILOGDEPS ${GEN_VLOG_DEPS})
       endif()
+
       if(${N_TEMPLATE_APPLY_VERILOG_IMAGE_GEN})
       endif()
     endforeach(SRC)
   endforeach(PREFIX)
 
-  add_custom_target(${N_TEMPLATE_NAME} ALL DEPENDS ${OUTPUTS})
+  add_custom_target(${N_TEMPLATE_NAME} ALL DEPENDS ${OUTPUTS} ${GEN_VLOG_DEPS} ${GEN_XML_DEPS})
```

Comment:
The OUTPUTS should already have transitive dependencies, so this shouldn't be required if things are working correctly.

Reply:
Without this the generated files included by other generated files were not included in the dependencies list

Reply:
I don't believe this statement does what you think it does.  If the "${GEN_VLOG_DEPS} ${GEN_XML_DEPS}" are required to build the file, this line does not help with that.  If these files are required when something else uses this file, this line does not help with that either.  What do you actually think this line does?